preview/MsixCore/msixmgr/msixmgr.rc

```diff
@@ -96,17 +125,32 @@ BEGIN
     IDS_STRING_REINSTALLAPP "Reinstall"
     IDS_RUNFULLTRUST_CAPABILITY "Uses all system resources"
     IDS_STRING_ERROR_MSG    "App installation failed with error message: An internal error occurred with error 0x"
-	IDS_STRING_HELPTEXT_DESCRIPTION " Description:\n -----------\n \tInstalls an msix package, removes an msix package or queries for msix packages.\n \tMay also unpack msix packages and apply ACLs to the resulting package folders\n\n"
+    IDS_STRING_HELPTEXT_DESCRIPTION 
+                            " Description:\n -----------\n \tInstalls an msix package, removes an msix package or queries for msix packages.\n \tMay also unpack msix packages and apply ACLs to the resulting package folders\n\n"
     IDS_STRING_HELPTEXT_OPTIONS " Options:\n -------\n"
-    IDS_STRING_HELP_OPTION_UNPACK "Unpack a package (.appx, .msix, .appxbundle, .msixbundle) and extract its contents to a folder."
-	IDS_STRING_HELP_OPTION_UNPACK_PACKAGEPATH "the path to the package to unpack"
-	IDS_STRING_HELP_OPTION_UNPACK_DESTINATION "the directory to place the resulting package folder(s) in"
-	IDS_STRING_HELP_OPTION_UNPACK_APPLYACLS "optional parameter that applies ACLs to the resulting package folder(s) and their parent folder"
-        IDS_STRING_HELP_OPTION_UNPACK_VALIDATESIGNATURE "optional parameter that validates a package's signature file before unpacking the package. This will require that the package's certificate is installed on the machine."
-	IDS_STRING_HELP_OPTION_APPLYACLS "Applies ACLs to a package folder (an unpacked package)"
-	IDS_STRING_HELP_OPTION_APPLYACLS_PACKAGEPATH "the path to the folder to apply acls to"
+    IDS_STRING_HELP_OPTION_UNPACK 
+                            "Unpack a package (.appx, .msix, .appxbundle, .msixbundle) and extract its contents to a folder."
+END
 
-	END
+STRINGTABLE
+BEGIN
+    IDS_STRING_HELP_OPTION_UNPACK_PACKAGEPATH 
+                            "the path to the package to unpack"
+    IDS_STRING_HELP_OPTION_UNPACK_DESTINATION 
+                            "the directory to place the resulting package folder(s) in"
+    IDS_STRING_HELP_OPTION_UNPACK_APPLYACLS 
+                            "optional parameter that applies ACLs to the resulting package folder(s) and their parent folder"
+    IDS_STRING_HELP_OPTION_UNPACK_VALIDATESIGNATURE 
+                            "optional parameter that validates a package's signature file before unpacking the package. This will require that the package's certificate is installed on the machine."
+    IDS_STRING_HELP_OPTION_APPLYACLS 
+                            "Applies ACLs to a package folder (an unpacked package)"
+    IDS_STRING_HELP_OPTION_APPLYACLS_PACKAGEPATH 
+                            "the path to the folder to apply acls to"
+    IDS_STRING_LOADING_FAILED "Loading Package Failed"
+    IDS_STRING_WXS_REMOVE   " Remove"

```

Comment:
>   IDS_STRING_WXS_REMOVE   " Remove" [](start = 1, length = 36)

if it's just "Remove", we don't need this localized.

Reply:
Won't Remove be different in different languages?

Reply:
Wix already has localization for "Remove", "Install" "Select the operation you wish to perform" etc all those strings in the dialog already; 
we just need localization for any non-standard custom strings we added.

---
In reply to: [337798141](https://github.com/microsoft/msix-packaging/pull/242#discussion_r337798141) [](ancestors = 337798141)

Reply:
Removed remove.